tests/integration_tests/performance/test_versioned_serialization_benchmark.py

```diff
@@ -36,14 +36,14 @@
             "crc": {"target": 0.108, "delta": 0.025},  # milliseconds  # milliseconds
         },
         "deserialize": {
-            "no-crc": {"target": 0.070, "delta": 0.02},  # milliseconds  # milliseconds
+            "no-crc": {"target": 0.030, "delta": 0.02},  # milliseconds  # milliseconds
```

Comment:
Are you sure? If yes, this is quite a nice improvement!

Reply:
It seems consistent.